Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -443,6 +447,7 @@ open class AVFoundationPlayback: Playback {
         }
 
         self.trigger(.airPlayStatusUpdate, userInfo: ["externalPlaybackActive": player!.isExternalPlaybackActive])
+        self.trigger(.didUpdateAirPlayStatus, userInfo: ["externalPlaybackActive": player!.isExternalPlaybackActive])
```

Comment:
Hey @dezouzart, maybe we can avoid the force unwrapping of `player` object. One option is to use `concretePlayer` object. What do you think?